config/config.yml

```diff
@@ -18,13 +18,13 @@ cacheBusterOn: 'true'
 exportDir: export
 publicDir: public
 sourceDir: source
-assetsDirDefault: '/assets'
+assetsDirDefault: /assets
 defaultPattern: introduction-intro
 defaultShowPatternInfo: true
 lineageMatch: '{%([ ]+)?include( |\()[&quot;\&#039;]([A-Za-z0-9-_]+)[&quot;\&#039;](\))?(.*)%}'
 lineageMatchKey: 3
 patternExtension: twig
-twigDebug: true
+twigDebug: false
```

Comment:
Muss das in den Master? 😱

Reply:
@pchr-srf mercivielmool! (ändere ich noch!)